addons/nuxeo-platform-document-routing/nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/io/TaskWriter.java

```diff
@@ -264,10 +266,13 @@ protected void writeWorkflowInitiator(JsonGenerator jg, String workflowInitiator
         jg.writeStringField("workflowInitiator", workflowInitiator);
     }
 
-    protected static ActionContext createActionContext(CoreSession session) {
+    protected static ActionContext createActionContext(GraphNode node, CoreSession session) {
```

Comment:
I would have deprecated the `createActionContext(CoreSession session)` method and add a new `createActionContext(GraphNode node, CoreSession session)`, especially if this code is to be backported on 9.10 and 10.10.